src/agents/agent.py

```diff
@@ -61,14 +62,14 @@ def __init__(self, base_model: str):
 
     def search_queries(self, queries: list, project_name: str) -> dict:
         results = {}
-        
+
         knowledge_base = KnowledgeBase()
-        bing_search = BingSearch()
+        bing_search = WebSearch()
```

Comment:
As we are using a common class for search engine we can rename bing_search to web_search